src/server/utils/validations.js

```diff
@@ -49,6 +49,14 @@ const validations = {
     }
     return validatedString;
   },
+
+  validateName: (data, count, nullable) => {
```

Comment:
`nameToUpperCase` wouldn't reflect what it's doing though, as it's still passing the data through to do the length and nullability checks. This function does also receive the 2nd and 3rd params (see line 430, for instance). I could consolidate the functions, but I think there's a use in keeping a separate `validateString` function.